log-collector/src/main/scala/io/phdata/pulse/logcollector/PulseKafkaConsumer.scala

```diff
@@ -48,8 +45,7 @@ class PulseKafkaConsumer(solrCloudStream: SolrCloudStream) extends JsonSupport w
         val records = consumer.poll(MAX_TIMEOUT)
         for (record <- records.asScala) {
           logger.trace("KAFKA: Consuming " + record.value() + " from topic: " + topic)
-          val logEvent    = record.value().parseJson.convertTo[LogEvent]
-          val logEventMap = Util.logEventToFlattenedMap(logEvent)
+          val logEventMap = record.value().parseJson.convertTo[Map[String, String]]
           solrCloudStream.put(logEventMap.getOrElse("application", ""), logEventMap)
```

Comment:
I don't think the 'orElse' here is valid. If there's no application name it's an invalid request and we should just throw it away. For now can you just add a TODO for proper error handling when there is no application name?

Reply:
@afoerster Where should I add the TODO? In Issue #99 ?

Reply:
I meant in the code, like // TODO ... (which can be searched easily), but actually adding it to the issue is a good idea

Reply:
added TODO in code and in Issue #99